src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -128,6 +128,12 @@ Copyright (c) .NET Foundation. All rights reserved.
                  FormatArguments="$(SdkResolverGlobalJsonPath)" />
   </Target>
 
+  <Target Name="_WarnWhenUsingNET6WithVS1611"
```

Comment:
This only checks if you're using VS 16, it doesn't check for specific minor versions, does it?

Reply:
Yeah, based on the warning text Kathleen suggested I assumed that was the desired behavior, @dsplaisted is that right or do we only want this with specifically 16.11?

Reply:
Yup, that's fine, I was more calling out that the target name contains "WithVS1611", when it's really just VS16

Reply:
Good catch Jacques, I think the target should be renamed.